WordPress/Classes/Models/Blog.m

```diff
@@ -458,6 +458,8 @@ - (BOOL)supports:(BlogFeature)feature
             return [self isHostedAtWPcom] && [self supportsSiteManagementServices];
         case BlogFeatureNoncePreviews:
             return [self supportsRestApi] && ![self isHostedAtWPcom];
+        case BlogFeatureMediaMetadataEditing:
+            return [self isHostedAtWPcom];
```

Comment:
I'm thinking this should be something like `[self supportsRestApi] && [self isAdmin];`? As we also lose support for self-hosted over Jetpack with just .com.